packages/mirinae/src/hooks/context-menu-controller/index.ts

```diff
@@ -37,14 +37,17 @@ export interface UseContextMenuControllerOptions {
 
     /* Required when to use show more button to attach items  */
     pageSize?: Ref<number|undefined>|number;
+
+    /* In the context of 'useFixedStyle,' to adjust the position of the context menu relative to the target, the default value is 'left.'  */
+    position?: string;
```

Comment:
suggestion: What about to give the specific type like 'left'|'top'|'bottom'|'right' ? :)